frontend/src/pages/my-projects/libs/helpers/init-debounce-helper.ts

```diff
@@ -0,0 +1,21 @@
+type FunctionToDebounce<T extends unknown[], R> = (...arguments_: T) => R;
+
+function initDebounce<T extends unknown[], R>(
```

Comment:
```suggestion
function initDebounce<T extends unknown[], R>(
```
prefer arrow function